packages/web-components/src/components/va-card/va-card.tsx

```diff
@@ -0,0 +1,41 @@
+import { 
+  Component,
+  Host, 
+  Prop, 
+  h 
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Card
+ * @maturityCategory dont_use
+ * @maturityLevel proposed
```

Comment:
I think that we can set this to "Use with Caution: Candidate" [like this other new component](https://design.va.gov/storybook/?path=/docs/components-va-need-help--default) we recently added:

```
 @maturityCategory caution
 @maturityLevel candidate
 ```